ee/clickhouse/models/property.py

```diff
@@ -254,6 +249,29 @@ def property_table(property: Property) -> TableWithProperties:
         raise ValueError(f"Property type does not have a table: {property.type}")
 
 
+def get_single_or_multi_property_string_expr(
+    breakdown, table: TableWithProperties, prop_var: str, identifier: Literal["prop", "value"]
+):
+    """
+    When querying for breakdown properties:
+     * If the breakdown provided is a string, we extract the JSON from the properties object stored in the DB
+     * If it is an array of strings, we extract each of those properties and concatenate them into a single value
+    clickhouse parameterizes into a query template from a flat list using % string formatting
+    values are escaped and inserted in the query here instead of adding new items to the flat list of values
+    """
+    if isinstance(breakdown, str) or isinstance(breakdown, int):
+        expression, _ = get_property_string_expr(table, str(breakdown), escape_param(breakdown), prop_var)
```

Comment:
I'm guessing `get_property_string_expr` doesn't do any escaping atm, should it have that responsibility instead?

Reply:
Normally it takes the name of a value to read from the filter params. That has to be a flat value. My first run through this I had to escape and pass in the breakdown. Let me review if that's still true...

Reply:
Ah, yep. ClickHouse does %formatting on the string. So we'd need to explode each value for breakdown into the data structure it formats from (breakdown_0 for Safari, breakdown_1 for Chrome etc). Those values if templated in by ClickHouse are escaped. I found that confusing so put them in directly here.